### PR TITLE
Ui/use tags apis

### PR DESCRIPTION
**Pull request by anwar6953**
Replace usages of context API with usages of tags API, for metrics.

https://issues.cask.co/browse/CDAP-2236

Note: it's not done yet, there's still a portion I labeled 'TODO: rethink', but the remainder can get reviewed in the meantime.

### CODE REVIEW

File: cdap-ui/app/features/datasets/controllers/tabs/status-ctrl.js [hunk 1]
```diff
@@ -41,8 +38,10 @@ angular.module(PKG.name + '.feature.datasets')
       }
 
       dataSrc.poll({
-        _cdapPath : '/metrics/query?metric=system.dataset.store.bytes&context=namespace.' +
-                    $state.params.namespace + '.dataset.' + currentDataset + '&aggregate=true',
+        _cdapPath : '/metrics/query?metric=system.dataset.store.bytes' +
```
Comment (ajainarayanan) [inline cdap-ui/app/features/datasets/controllers/tabs/status-ctrl.js]: Can we use ```myHelpers.tagsToParams``` ?

**Comment by ajainarayanan** [comment]
One minor comment. Apart from that :+1: LGTM.

File: cdap-ui/app/features/datasets/controllers/tabs/status-ctrl.js (hunk 1 above)
Comment (anwar6953) [inline cdap-ui/app/features/datasets/controllers/tabs/status-ctrl.js]: done (on the following line).

**Comment by ajainarayanan** [comment]
:+1: LGTM

File: cdap-ui/app/features/flows/controllers/tabs/runs/flowlets/output-ctrl.js [hunk 2]
```diff
@@ -24,15 +32,12 @@ angular.module(PKG.name + '.feature.flows')
...
-                            + '.app.' + $state.params.appId
-                            + '.flow.' + $state.params.programId
-                            + '.run.' + runid
-                            + '.flowlet.' + flowletid
+              _cdapPath: '/metrics/query?' + myHelpers.tagsToParams(flowletTags)
                             + '&metric=system.process.events.out&start=now-60s&count=60',
               method: 'POST'
             }, function(res) {
               if (res.series[0]) {
+                debugger;
```
Comment (gsps1) [inline cdap-ui/app/features/flows/controllers/tabs/runs/flowlets/output-ctrl.js]: remove ?

Comment (anwar6953) [inline cdap-ui/app/features/flows/controllers/tabs/runs/flowlets/output-ctrl.js]: its been removed on develop, already.

**Pull request merged**